### DISCUSSION TITLE
eclccpath - Property is not allowed

### DISCUSSION THREAD

**Discussion by tr3ntbuck** [hpcc-systems/vscode-ecl#231] [Q&A] [answered]
How can we specify which compiler to use?   eclccpath is not allowed in launch.json file

and it does not seem to have an effect in settings.json

**Comment by GordonSmith** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/966863/107369548-d5a8f780-6ad9-11eb-8efd-948721f8cb1a.png)

**Reply by GordonSmith** [reply]
1 - Click on the compiler version in the status bar.
2 - Select which version you would prefer to use,

**Comment by tr3ntbuck** [comment]
Well, that was easy.   Thanks

1 - Click on the compiler version in the status bar.
2 - Select which version you would prefer to use,

**Reply by GordonSmith** [reply]
In general you will want to pick "auto" and install any missing Client Tools so the IDE can auto pick the best version for the targeted platform.
